### PR TITLE
Check code 4

### PR DESCRIPTION
**Pull request by harsh-groverfk**

### CODE REVIEW

**Review by alekh2 — APPROVED** [review]

**Review by dan13ram — COMMENTED** [review]
This code is  bad!!

**Comment by dan13ram** [comment]
asdkjsahdhaskd

**Review by swagatata — COMMENTED** [review]
sdflskdjflksjf

**Comment by swagatata** [comment]
sdjflskdjf

File: chec_code_commit2.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print "Check code commit 2"
+
```
Comment (dan13ram) [inline chec_code_commit2.py:2]: This looks good

**Pull request merged**